docs/index.md

```diff
@@ -0,0 +1,9 @@
+# The Energy Exascale Earth System Model (E3SM)
+
+The documentation for the components of E3SM is found here.
+
+## Components
+- [ELM](./ELM/index.md)
```

Comment:
My guts tell me that the path here should be something like `../components/elm/docs/index.md`. Did you try this locally (via `mkdocs serve`), and verify it renders correctly?

Reply:
Nvm, I just read your comment on the PR page now.

Reply:
yes that and the monorepo plugin.  https://github.com/backstage/mkdocs-monorepo-plugin/tree/master